students/chieu_quach/lesson08/circle.py

```diff
@@ -0,0 +1,159 @@
+
+import math
+
+class Circle(object):
+ 
+    def __init__(self, radius):
+
+        self.radius = radius
+
+    @property
+    def get_diameter(self):
+   # def get_diameter(self, value):
+      
+        return self.radius * 2   
+
+    @get_diameter.setter
+    def get_diameter(self, value):
```

Comment:
note that in directions this was called `diameter` (properties are usually nouns)